seeder/postgres_seeder/data_generator/config.js

```diff
@@ -20,10 +20,10 @@ Configurable options:
 */
 
 const config = {
-  product_range: 10,
+  product_range: 95239,
```

Comment:
**Annotation:** 95,239 products equate to 10+ million records as it branches into 7 different styles and each style branches off into 15 or 17 sizes depending on gender.